OJP/OJP_Trips.xsd

```diff
@@ -174,23 +179,66 @@
 			<xs:documentation>the types of algorithm that can be used for planning a journey (fastest, least walking, etc). If multiple optimisation methods are needed, then several requests should be performed in parallel.</xs:documentation>
 		</xs:annotation>
 		<xs:restriction base="xs:string">
-			<xs:enumeration value="fastest"/>
-			<xs:enumeration value="minChanges"/>
-			<xs:enumeration value="leastWalking"/>
-			<xs:enumeration value="leastCost"/>
+			<xs:enumeration value="fastest">
+				<xs:annotation>
+					<xs:documentation>Shortest duration.</xs:documentation>
```

Comment:
```suggestion
					<xs:documentation>Shortest duration somewhere in the future. This may present a shorter trip than the next earliest arrival (with latest departure).</xs:documentation>
```